redis_enterprise/assets/dashboards/redis_enterprise_proxy.json

```
@@ -1,4 +1,4 @@
-{
+ {
```

```suggestion
{
```
Reverting this change because it does not concern the `redis_enterprise_prometheus` integration